app/assets/javascripts/templates/visualizations/controls/_checkbox.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="checkbox">
+  <input id="ckbx-{{id}}" type="checkbox" data-log-id="{{logId}}"
+         value="{{value}}" name="{{name}}"/>
+  <label for="ckbx-{{id}}" style="{{style}}" class="name">{{label}}</label>
+  {{#if picker}}
+    <a href="#" class="color-picker"
+       data-color="{{picker.color}}" data-color-id="{{value}}">
+      <img class="cpickr-icon" height="20px" width="20px"
+           src="{{picker.icon}}">
+    </a>
+  {{/if}}
+</div>
```

Comment:
The "style" specification is dependent on specific parts of the vis which seemed to be the cleanest way to do this at the time, but I will update the height and width.